src/main/java/org/candlepin/subscriptions/metering/MeteringEventFactory.java

```diff
@@ -127,9 +127,9 @@ public static void updateMetricEvent(
         .withRole(getRole(role, accountNumber, instanceId));
   }
 
-  public static String getEventType(String metricId) {
-    return StringUtils.hasText(metricId)
-        ? String.format("%s_%s", EVENT_TYPE, metricId)
+  public static String getEventType(String rhMarketplaceMetricId) {
+    return StringUtils.hasText(rhMarketplaceMetricId)
+        ? String.format("%s_%s", EVENT_TYPE, rhMarketplaceMetricId)
         : EVENT_TYPE;
   }
 
```

Comment:
Actually for MeteringEventFactory, we should keep it as `metricId`. Unfortunately, we overloaded `metricId` at this point. Sometimes its a canonical identifier for a metric; sometimes its the RHM-specific identifier. Please revert changes in this file.

Reply:
Alright will do